spec/game_spec.rb

```diff
@@ -0,0 +1,27 @@
+require_relative '../classes/game'
+require_relative '../classes/item'
+
+describe Game do
+  before :each do
+    @game = Game.new(true, 2021, 2002, false)
+    @game2 = Game.new(true, 2019, 2002, false)
+    # @item = Item.new(2002, false)
```

Comment:
You can remove this commented line and remove require_relative Item

Reply:
OK,
Thank You!